### PR TITLE
Enable tests while deploying

### PR DESCRIPTION
**Pull request by Matthew-Kulich**

### CODE REVIEW

File: .github/workflows/protect-master.yml [hunk 1]
```diff
@@ -23,9 +23,9 @@ jobs:
     - uses: actions/checkout@v2
 
     # Runs a single command using the runners shell
-#    - name: Compile
-#      run: mvn compile
+    - name: Compile
```
Comment (blcham) [inline .github/workflows/protect-master.yml:26]: Not sure if we need "mvn compile". If not it would be very good to not include it.

**Review by blcham — APPROVED** [review]
LGTM, but see my comment about "mvn compile"

File: .github/workflows/protect-master.yml (hunk 1 above)
Comment (Matthew-Kulich) [reply .github/workflows/protect-master.yml:26]: It looks like, we don't need to compile

Comment (Matthew-Kulich) [reply .github/workflows/protect-master.yml]: Removed

**Pull request merged**